datasources.tf

```
@@ -0,0 +1,15 @@
+data "azurerm_resource_group" "example" {
+  name = "ilyakhova"
+}
+
+resource "azurerm_storage_account" "example" {
```

Не варто називати усі ресурси `example`
```suggestion
resource "azurerm_storage_account" "storage_account_dafault" {
```